NAMESPACE

```diff
@@ -2,4 +2,12 @@
 
 export(analyse_oneshot_spatial)
 export(analyse_rasters_spatial)
+export(basedata)
+export(install_perlbrewr)
+export(load_data_)
+export(process_tabular_results)
+export(run_cluster_analysis)
+export(run_randomisation_analysis)
+export(run_spatial_analysis)
 export(start_server)
+importFrom(perlbrewr,perlbrew)
```

Comment:
Probably for later, but we only need perlbrewr on non-Windows machines.  We have Strawberry perl on Windows, and perlbrew does not work on Windows.

Reply:
Unfortunately, R packages does not allow OS specific dependencies. I think this is ok for now, and perlbrewr is pretty lite